src/resources/js/Pages/Employee/Logs/Index.vue

```diff
@@ -137,18 +151,23 @@ export default {
 
   data() {
     return {
-      searchTerm: '',
+      filterByLogType: '',
       logTypes,
     };
   },
   computed: {
     filteredLogs() {
-      if (!this.searchTerm) return this.logs;
-      return this.logs.filter((log) => {
-        return log.action == this.searchTerm;
-      });
+      if (!this.filterByLogType) return this.logs;
+      window.location.href = this.url + '?filterByLogType=' + this.filterByLogType;
+      return false;
     },
   },
   methods: { toTitleCase },
 };
+
+window.onload = function(){
```

Comment:
Could you refactor this to be more idomatic Vue? Both Vue and React have more recommended ways to do things onload

Reply:
This change has been made.